src/azure-account.ts

```diff
@@ -387,10 +387,23 @@ export class AzureLoginHelper {
 						ignoreFocusOut: true
 					});
 					if (armUrl) {
-						await config.update(customCloudArmUrlKey, armUrl, getCurrentTarget(config.inspect(customCloudArmUrlKey)));
+						const tenantId = await window.showInputBox({
+							prompt: localize('azure-account.enterTenantId', "Enter the Tenant Id"),
+							placeHolder: "Tenant Id..",
```

Comment:
"Tenant id", extra period here

Reply:
this should also be localized

Reply:
Do we need to localized the placeholder?